cli/src/app.rs

```diff
@@ -88,11 +88,6 @@ pub fn add_subcommands(command: Command) -> Command {
                 Arg::new("JOB_ID")
                     .value_name("JOB_ID")
                     .help("The job id to query (or `current` for the most recent job)"),
-                Arg::new("verbose")
-                    .action(ArgAction::SetTrue)
-                    .short('v')
-                    .long("verbose")
```

Comment:
The global argument only has the short form and not the long form. So calls like this will now fail:

```
phylum analyze --verbose --json
```

And I think that is exactly how `phylum-ci` runs that command... Pinging @maxrake to confirm.

Reply:
I already have a comment on a draft review in place saying exactly this. You beat me to it!

Reply:
We can just add the long form tot he global command, no issue there.